### PR TITLE
Transfer utils into polkadotcloud/core

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +64,12 @@
     "typescript": "^5.1.6"
   },
   "workspaces": [
-    "packages/*",
+    "packages/cloud-core",
```
Comment (rossbulat) [inline package.json:67]: The glob pattern should be fine, any reason why this was changed? Manually doing this become hard to manage as more packages are added

File: packages/cloud-core/rollup.config.mjs [hunk 2]
```diff
@@ -11,16 +11,16 @@ import cleanup from "rollup-plugin-cleanup";
  * @type {import('rollup').RollupOptions}
  */
 export default {
-  input: "lib/index.ts",
+  input: "lib/utils/index.ts",
   output: [
     {
-      file: "dist/index.ts",
+      file: "dist/utils/index.ts",
```
Comment (rossbulat) [inline packages/cloud-core/rollup.config.mjs:17]: Shall we change output to `.js`? should still build with accompanying `.d.ts` file.

Comment (wirednkod) [reply packages/cloud-core/rollup.config.mjs:17]: Oh my.. that went totally under my radar. Thank you

File: package.json (hunk 1 above)
Comment (rossbulat) [reply package.json:67]: Ok, if it becomes troublesome in the future we can revisit & enhance the build process.

**Review by rossbulat — APPROVED** [review]

**Pull request merged**